ocs_ci/deployment/aws.py

```diff
@@ -119,6 +135,10 @@ def deploy_ocp(self, log_cli_level='DEBUG'):
             log_cli_level (str): openshift installer's log level
                 (default: "DEBUG")
         """
+        if self.check_cluster_existence() and not (
+            config.DEPLOYMENT.get('force_deploy_multiple_clusters')
+        ):
+            raise SameNamePrefixClusterAlreadyExistsException
```

Comment:
Do you think it's worth to put some message to the exception or warning msg in function is enough?
Thinking that we could design this function to raise exception in case of exist and call that function only when `not config.DEPLOYMENT.get('force_deploy_multiple_clusters')` . WDYT?

Reply:
done